test/test_integration.py

```diff
@@ -382,8 +382,8 @@ def socket_handler(listener):
             # We should get two packets: one connection header string, one
             # SettingsFrame. Rather than respond to the packets, send a GOAWAY
             # frame with error code 0 indicating clean shutdown.
-            first = sock.recv(65535)
-            second = sock.recv(65535)
+            first = sock.recv(65535)  # pragma: no flakes
```

Comment:
Does pytest-flakes add this? `# prgama: no flakes` is not something included in flake8. Anyone running flake8 normally on this will still see this error.